packages/ui/cart/drawer/CartLineItem.tsx

```diff
@@ -20,7 +20,7 @@ import {
    useUpdateLineItemQuantity,
 } from '@ifixit/cart-sdk';
 import { motion } from 'framer-motion';
-import { IfixitImage } from '@components/ifixit-image';
+import { IfixitImage } from '../../misc/IfixitImage';
```

Comment:
For this import change, why are we importing from a path instead of `@ifixit/ui` like in the other changes?

Reply:
Good question. `CartLineItem` also lives inside `@ifixit/ui`, and isn't aware that the name of the package it lives in is `@ifixit/ui`, so we use relative paths instead. See another similar example here:

https://github.com/iFixit/react-commerce/blob/39d5aa6e67d1b6211d6d02b025114653b04547df/packages/ui/cart/drawer/CartDrawer.tsx#L42-L43

https://github.com/iFixit/react-commerce/blob/39d5aa6e67d1b6211d6d02b025114653b04547df/packages/ui/homepage-kpis/index.tsx#L9